recipes/samarium/all/conanfile.py

```diff
@@ -12,17 +16,15 @@ class SamariumConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index/"
     license = "MIT"
     topics = ("cpp20", "physics", "2d", "simulation")
+    generators = "CMakeDeps", "CMakeToolchain"
+    requires = "fmt/9.0.0", "sfml/2.5.1", "range-v3/0.12.0", "stb/cci.20210910", "tl-expected/20190710"
 
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
+    options = {"shared": [True, False], "fPIC": [
+        True, False]}
     default_options = {"shared": False, "fPIC": True}
 
-    generators = "cmake", "cmake_find_package_multi"
-    requires = "fmt/8.1.1", "sfml/2.5.1", "range-v3/0.12.0"
-
-    @property
-    def _min_cppstd(self):
-        return "20"
+    exports_sources = "patches/*"
```

Comment:
We have new preference for this https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#exporting-patches which would be advantage for the supported versions